src/app/core/services/record-api.service.ts

```diff
@@ -58,7 +59,9 @@ export class RecordApiService extends CommonApiService {
     this.currentRecordApiUrl = `${apiUrl}/${pidType}/${pidValue}`;
     this.currentRecordEditorApiUrl = `${editorApiUrl}/${pidType}/${pidValue}`;
     this.newRecordFetched$.next(null);
-    return this.fetchUrl(this.currentRecordApiUrl);
```

Comment:
Modify `https://github.com/inspirehep/record-editor/blob/42089b71478623a97abdd21abd6636f2fc8f221c/src/app/core/services/common-api.service.ts#L38` to accept `headers` instead.